new/LIT6351MEMiracle.xml

```diff
@@ -0,0 +1,106 @@
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6351MEMiracle" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Miracle of ʾEwosṭātewos: the woman who stopped the sun to finish her daily work before the Sabbath</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MV"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MV" when="2021-01-20">Created entity</change>     
+            <change who="MV" when="2021-01-21">Added bibliography, fixed title</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listBibl type="editions">
+                    <bibl>
+                        <ptr target="bm:Turaiev1905Eustathii"/>
+                        <citedRange unit="page"><!-- insert page range from Turaiev's Monumenta Aethiopiae Hagiologica: Vita et Miracula Eustathii, pp. 134-177 --> (2nd miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+                <listBibl type="translation">
+                    <bibl>
+                        <ptr target="bm:Turaev1906Ewostatewos"/>
+                        <citedRange unit="page">72-73 (2nd miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+                <listBibl type="secondary">
+                    <bibl>
+                        <ptr target="bm:GrebTiss1935Codices"/>
```

Comment:
You could add an alternative title as given in this catalogue, if we think this could be helpful for the catalogue

I did it for the Miracles of george, e.g., https://betamasaheft.eu/works/LIT6209MGAndrew/main